core/admin/admin-notices.php

```diff
@@ -428,6 +429,85 @@ private function site_has_woocommerce() {
 		return class_exists( 'WooCommerce' );
 	}
 
+	private function get_installed_form_plugin_name() {
+		$form_plugins = [
```

Comment:
Do we need to define the arrays on every entry to this function?
Also, please change to a meaningful name. maybe form_plugins_contancts_to_name_mapper
same of the second array 

maybe extract to smaller functions 
name = try_get_name_by_constant
if name is still null
name = try_get_name_by_class  
if name is still null return 'form'